tests/util.coffee

```diff
@@ -1,2 +1,5 @@
 exports.loadEnv = ->
 	require('dotenv').config(silent: true)
+
+exports.titleCase = (str) ->
+	str.replace(/\b(\w)/g, (w) -> w.toUpperCase())
```

Comment:
Does lodash not have a function for this?

Reply:
I think not and that's why I got this from the UI.
There is [startCase](http://devdocs.io/lodash~4/index#startCase) which isn't 100% the same but could work here.

Reply:
Whatever's simplest, it's not a big deal either way.